libs/kano-profile/includes/kano_profile_bindings.h

```diff
@@ -29,12 +29,25 @@ namespace kano_profile {
     {
         public:
             apps();
+            void finalise();
```

Comment:
That's true. However, the reason behind the re-implementation was to have a place where the class would do more clean up, otherwise the method in the `Binding` class would be called. I wrote it like that to make that explicit, so if there'd come a time to do more clean up, you'd just throw things in the re-implemented method. I don't mind, I can change it if you'd like.